litex_boards/platforms/pano_logic_g2.py

```diff
@@ -97,6 +97,9 @@
         Subsignal("cke", Pins("D2"), IOStandard("SSTL18_II")),
         Subsignal("odt", Pins("J6"), IOStandard("SSTL18_II")),
     ),
+    # Ethernet phy reset (clk125 is 25 Mhz instead of 125 Mhz if reset is active)
```

Comment:
Can you add a link to the documentation about this?

Reply:
Done.